src/Language/PureScript/Docs/Collect.hs

```diff
@@ -103,8 +103,18 @@ compileForDocs outputDir inputFiles = do
 
   where
   renderProgressMessage :: P.ProgressMessage -> String
-  renderProgressMessage (P.CompilingModule mn) =
-    "Compiling documentation for " ++ T.unpack (P.runModuleName mn)
+  renderProgressMessage (P.CompilingModule mn mi) =
+    concat
+      [ renderProgressIndex mi
+      , "Compiling documentation for "
+      , T.unpack (P.runModuleName mn)
+      ]
+    where
+    renderProgressIndex = maybe "" $ \(start, end) ->
+      let start' = show start
+          end' = show end
+          preSpace = replicate (length end' - length start') ' '
+      in "[" <> preSpace <> start' <> " of " <> end' <> "] "
```

Comment:
Since this is the same function as used for the corresponding `renderProgressIndex` when compiling modules, should this code be defined once and used in both places?